docs/topics/scheduler.rst

```diff
@@ -0,0 +1,59 @@
+.. _topics-scheduler:
+
+=========
+Scheduler
+=========
+
+The Scheduler receives requests from the :ref:`Engine <component-engine>`
+and enqueues them into disk and/or disk queues
+(:meth:`scrapy.core.scheduler.Scheduler.enqueue_request`).
+It also gets those requests and feeds them back when the engine requests
+them (:meth:`scrapy.core.scheduler.Scheduler.next_request`).
+
+Overriding the default Scheduler
+================================
+
+You can use your own custom `Scheduler` class by supplying the full
+Python path in the :setting:`SCHEDULER` setting
+
+
+Scheduler interface
+===================
+
+This is the basic Scheduler API, i.e. the methods that are defined in the default Scheduler.
+
+.. module:: scrapy.core.scheduler
+
+.. autoclass:: Scheduler
```

Comment:
I think we need to make a clear distinction between the API that a scheduler class must provide, and the API of the default scheduler class.

Reply:
Done in f27946a8d96f7b6958f203a3c45b710b316a10cd